main.nf

```diff
@@ -2945,18 +2962,25 @@ process Mpileup {
         file(fastaFai) from ch_fai
 
     output:
-        set idPatient, idSample, file("${prefix}${idSample}.pileup.gz") into mpileupMerge
+        set idPatient, idSample, file("${prefix}${idSample}.pileup") into mpileupMerge
 
     when: 'controlfreec' in tools || 'mpileup' in tools
 
     script:
     prefix = params.no_intervals ? "" : "${intervalBed.baseName}_"
     intervalsOptions = params.no_intervals ? "" : "-l ${intervalBed}"
+
+    if(params.normal_pileup && params.tumor_pileup)   // we already have pileups
+    """
+    touch ${prefix}${idSample}.pileup 
```

Comment:
Not sure I understand the idea here

Reply:
So instead of a useless process here and in the next one, I think we make a new channel that match the ControlFreec process with the input mpileups and make it conditional so we don't even use these two processes at all.
That would be better, and safer, and easier to understand